schedule/functional/sle16/agama-create-hdd-textmode.yaml

```diff
@@ -7,6 +7,7 @@ schedule:
     - yam/agama/agama_auto
     - installation/grub_test
     - installation/first_boot
+    - console/validate_repos
```

Comment:
when modifying schedule from other squads you need to notify them, they also will need to setup test data, so better do it for our squad and when you have done it for all architecture, you could share it if you want.

Reply:
@okynos, please drop this change so other squad can apply this, test data need to be duplicated because other squad shoult not depend on test data under yam/ and functional/ folder is not qe security specific. Let each squad apply the change.

Reply:
Done